dts/arm/nordic/nrf54l20_cpuapp.dtsi

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
dt should come before soc content, otherwise soc Kconfig options can't be used at all (ref: keep things bisectable)

Reply:
done